examples/terragrunt/README.md

```diff
@@ -0,0 +1,49 @@
+# Terragrunt
+
+This example shows how to run Infracost Azure Devops tasks with Terragrunt.
+
+[//]: <> (BEGIN EXAMPLE)
+```yml
+pr:
+  - master
+
+variables:
+  API_KEY: $(apiKey)
+  
+jobs:
+  - job: terragrunt
+    displayName: Terragrunt
+    pool: 
+      vmImage: ubuntu-latest
+
+    steps:
+      - task: TerraformInstaller@0
+        displayName: Install Terraform
+
+      - bash: |
+          INSTALL_LOCATION="$(Build.SourcesDirectory)/bin"
+          mkdir -p ${INSTALL_LOCATION}
+          
+          LATEST_URL=$(curl -sL  https://api.github.com/repos/gruntwork-io/terragrunt/releases  | jq -r '.[0].assets[].browser_download_url' | egrep 'linux.*amd64' | tail -1)
```

Comment:
Should we use this URL `https://github.com/gruntwork-io/terragrunt/releases/latest/download/terragrunt_linux_amd64`? The GitHub API has rate limits for anonymous requests so might fail without an GitHub token in the header at times.

Reply:
yup - I'll update

Reply:
resolved